### ISSUE TITLE
Easier way to set graph for many cores CPU

### ISSUE THREAD

**Issue by XuebingZhao** [amanusk/s-tui#188]
**Is your feature request related to a problem? Please describe.**
I run s-tui on a 96c192t machine, It's painful to turn off per-core displays.

**Describe the solution you'd like**
Set the default diskplay to per-pakage for cores > i.e. 20

**Additional context**
Add any other context  about the feature request here. (screenshots, links to similar solutions)

**Comment by amanusk** [comment]
I agree. 96core machines where not in my mind when getting started :) 

Perhaps a toggle of avg/per-core should be an easy enough feature to implement, to easily reduce noise levers
